web/src/main/java/ua/com/vertex/utils/Aes.java

```diff
@@ -19,23 +19,23 @@ private static String pad2(String n) {
     }
 
     private static String hex(byte[] bytes) {
-        String r = "";
+        StringBuilder r = new StringBuilder();
```

Comment:
Sorry, but this is not my implementation. I do not know the exact purpose of this variable ((